content/installation/node/NodeSupportedTechnologies.md

```diff
@@ -4,30 +4,44 @@ description: "List of supported technologies"
+
+> **Note:** The Node.js agent currently doesn't support HTTP2, and generally doesn't support Node.js features classified as Experimental (Stability: 1).
+
+## Web Framework Support
+
+* [Express](http://expressjs.com) version 4
+* [hapi](https://www.npmjs.com/package/hapi) versions 16 and 17
+* [Sails](http://sailsjs.org/) version 0.12
+
+While the agent can still run on web frameworks that aren't officially supported, Contrast may produce less-specific findings than it would for supported frameworks. Instead of reporting that a vulnerability occurs in your application code, Contrast may need to report it within the framework code where it interfaces directly with Node.js's built-in [http](https://nodejs.org/api/http.html) module.
+
+
+## Third-Party Module Support
+
+* [express-session](https://github.com/expressjs/session)
+* [MongoDB](https://docs.mongodb.org/ecosystem/drivers/node-js/)
+* [MySQL](https://www.npmjs.com/package/mysql)
+* [Mongoose](http://docs.sequelizejs.com/)
+* [postgres](https://www.npmjs.com/package/pg)
+* [Sequelize](http://docs.sequelizejs.com/)
 
 Contrast doesn't guarantee support for old or deprecated versions of third-party modules.
 
 ## OS Support
 
-The agent runs entirely in the Node.js application layer with no *C* dependencies. As a result, it works on **all of the same operating systems as Node.js**, including Linux, Windows, macOS and Unix.
+The agent runs entirely in the Node.js application layer with no *C* dependencies. As a result, it works on **the same operating systems as Node.js**, including Linux, Windows, macOS and Unix.
 
 ## Testing Environments
 
-When changes are made, Contrast runs a battery of automated tests to ensure that it detects findings in supported technologies across all supported versions of Node. These automated tests exercise the Contrast agent against Contrast's [Node Test Bench](https://github.com/Contrast-Security-OSS/NodeTestBench) application. This application is updated as more third-party library support is added to the agent. 
+When changes are made, Contrast runs a battery of automated tests to ensure that it detects findings in supported technologies across all supported versions of Node. This includes tests that exercise the agent against Contrast's
+[Node Test Bench](https://github.com/Contrast-Security-OSS/NodeTestBench), [Hapi 16 Test Bench](https://github.com/Contrast-Security-OSS/Hapi16TestBench), and [Hapi 17 Test Bench](https://github.com/Contrast-Security-OSS/HapiTestBench) applications. Each of these applications is updated as Contrast adds more third-party library support to the agent.
 
-> If you want to add test cases, let Contrast know by submitting a pull request. 
+If you want to add test cases, let Contrast know by submitting a pull request.
```

Comment:
To where/which repo should they submit a PR?

Reply:
Whichever app they'd want to change they'd submit the PR to its repository. The use of the words "test case" here means any code changes they'd want to submit in order to test the agent's compatibility against them. Do you find the wording confusing or unclear of its meaning?

Reply:
@meandthemoon It's not terribly confusing. I think adding something simple like "by clicking the link for your chosen application and submitting a pull request" might clear it up.